src/scripts/deploy.sh

```diff
@@ -7,17 +7,20 @@ SOURCE_BRANCH="master"
 # Pull requests and commits to other branches shouldn't try to deploy, just build to verify
 if [ "$TRAVIS_PULL_REQUEST" != "false" -o "$TRAVIS_BRANCH" != "$SOURCE_BRANCH" ]; then
     echo "Skipping deploy; just doing a build and linting links/prose/js."
-    npm run build
-    npm test
+    yarn test
+    yarn build
```

Comment:
I hope the change in order between test and build doesn't break anything.

Reply:
Why should it break anything? On the other hand, we want to prevent build if tests are failing